man/kernel-command-line.xml

```diff
@@ -408,6 +408,16 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>systemd.default_device_timeout_sec=</varname></term>
```

Comment:
do we really this is as kernel cmdline arg? I mean, I am not totally opposed. But I am wondering: do you need this, do you want to actually make use of this? Because, if this is just for "completeness", I am not convinced, then we shouldn't have it, and maybe add later when somebody actually asks.

Having less knobs exposed is a good thing, hence let's try to be more minimal with these things.

Reply:
Both have been asked for, but without any explanation why the cmdline option is also needed. I can think of only one reason for it myself: if one faces a system that doesn't boot because of block device timeouts, it's handy to have an easy way to get it to boot (and then configure it properly). I'm not sure if that's good enough reason though.

Reply:
I've dropped the cmdline option.